Kubernetes/flannel/l2bridge/cni/config/cni.conf

```diff
@@ -1,45 +1,45 @@
-                                                "Value":  {
-                                                              "Type":  "ROUTE",
-                                                              "DestinationPrefix":  "10.124.24.33/32",
-                                                              "NeedEncap":  true
-                                                          }
-                                            }
-                                        ]
-                 }
-}
+{
+    "cniVersion":  "0.2.0",
+    "name":  "cbr0",
+    "type":  "flannel",
+    "delegate":  {
+                     "type":  "l2bridge",
+                     "dns":  {
+                                 "Nameservers":  [
+                                                     "10.96.0.10"
+                                                 ],
+                                 "Search":  [
+                                                "svc.cluster.local"
+                                            ]
+                             },
+                     "AdditionalArgs":  [
+                                            {
+                                                "Name":  "EndpointPolicy",
+                                                "Value":  {
+                                                              "Type":  "OutBoundNAT",
+                                                              "ExceptionList":  [
+                                                                                    "10.244.0.0/16",
+                                                                                    "10.96.0.0/12",
+                                                                                    "10.127.130.0/24"
+                                                                                ]
+                                                          }
+                                            },
+                                            {
+                                                "Name":  "EndpointPolicy",
+                                                "Value":  {
+                                                              "Type":  "ROUTE",
+                                                              "DestinationPrefix":  "10.96.0.0/12",
+                                                              "NeedEncap":  true
+                                                          }
+                                            },
+                                            {
+                                                "Name":  "EndpointPolicy",
+                                                "Value":  {
+                                                              "Type":  "ROUTE",
+                                                              "DestinationPrefix":  "10.127.130.36/32",
+                                                              "NeedEncap":  true

```

Comment:
Make it clear that this is needed for... whatever it's needed for.

Reply:
These are configurable and will all be overwritten by the script. This is done once by specify them in the start.ps1 script:
.\start.ps1 -ManagementIP <Windows_Worker_Mgmt_IP> -ClusterCIDR -ServiceCIDR -KubeDnsServiceIP